backend/apps/content/models.py

```
@@ -49,6 +49,18 @@ class Meta:
 
 
+class LessonVersion(models.Model):
+    lesson = models.ForeignKey(
+        Lesson, related_name="versions", on_delete=models.CASCADE
+    )
+    content = models.TextField()
+    summary = models.TextField(blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ["-created_at"]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)